homeassistant/components/shelly/climate.py

```diff
@@ -210,7 +210,7 @@ def available(self) -> bool:
         """Device availability."""
         if self.device_block is not None:
             return not cast(bool, self.device_block.valveError)
-        return self.coordinator.last_update_success
+        return super().available
```

Comment:
Not directly related to this PR, but reviewing the code, trying to understand why climate has different `available` property I realized this is probably incorrect.

A valve error should not mark the device unavailable (since the device is available and communicating) but we probably need to create a "valve error" sensor .